Kitodo/src/main/resources/messages/messages_es.properties

```diff
@@ -1083,6 +1083,7 @@ taskManagerIsRunning=No iniciar las tareas automáticamente
 taskManagerIsStopped=Inicie las tareas automáticamente
 taskManager=Gestor de tareas
 taskSaving=La tarea está guardada...
+taskScriptThread=Aŭtomata tasko
```

Comment:
Taking a look at the Spanish words for `task` in the previous and next lines of this file gives the impression that "tasko" may not be the best translation here. Since @danilopenagos kindly offered to contribute the Spanish translation for new messages, maybe you could just leave this line out?

Reply:
I clicked next to it and got Esperanto instead of Español. I think, should be: Tarea automática